.github/workflows/reviewdog.yml

```diff
@@ -0,0 +1,22 @@
+---
+# SPDX-FileCopyrightText: 2020 James Mills / prologic
+#
+# SPDX-License-Identifier: MIT
+
+name: reviewdog
+on: [pull_request]
+jobs:
+  remark-lint:
+    name: runner / remark-lint
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
```

Comment:
should use checkout@v3. checkout@v2 depends on a deprecated node.js version (see gh actions logs)